crates/ra_assists/src/handlers/remove_dbg.rs

```diff
@@ -126,4 +140,52 @@ fn foo(n: usize) {
             "dbg!(n.checked_sub(4))",
         );
     }
+
+    #[test]
+    fn test_remove_dbg_keep_semicolon() {
+        // https://github.com/rust-analyzer/rust-analyzer/issues/5129#issuecomment-651399779
+        // not quite though
+        // adding a comment at the end of the line makes
+        // the ast::MacroCall to include the semicolon at the end
+        let code = "
+let res = <|>dbg!(1 * 20); // needless comment
+";
+        let expected = "
+let res = 1 * 20; // needless comment
+";
+        check_assist(remove_dbg, code, expected);
```

Comment:
Let's inlice code & expected variables, and also make them raw string. That way, rust-analyzer will syntax-highlighth them, and it's the style we use for most assits tests.